pkg/tracker/stamped/tracker.go

```diff
@@ -12,17 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tracker
+package stamped
 
-//go:generate go run -modfile ../../hack/tools/go.mod github.com/maxbrunsfeld/counterfeiter/v6 -generate
+//go:generate go run -modfile ../../../hack/tools/go.mod github.com/maxbrunsfeld/counterfeiter/v6 -generate
 
 import (
 	"github.com/go-logr/logr"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 )
 
-//counterfeiter:generate . DynamicTracker
-type DynamicTracker interface {
-	Watch(log logr.Logger, obj runtime.Object, handler handler.EventHandler) error
+//counterfeiter:generate . StampedTracker
+type StampedTracker interface {
+	Watch(log logr.Logger, stampedObj runtime.Object, handler handler.EventHandler) error
```

Comment:
small collision when we update cluster-api - https://github.com/vmware-tanzu/cartographer/pull/596

Reply:
rebased, thanks!